manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -184,3 +184,26 @@ spec:
       for: 3m
       labels:
         severity: critical
+    - alert: etcdDivergentRevisions
+      annotations:
+        description: etcd is reporting divergent member revisions, with a pod
+          {{ $labels.pod }} on instance {{ $labels.instance }} {{ $value }}
+          compacted revisions behind pod {{ $labels.max_pod }} on instance
+          {{ $labels.max_instance }}. Members may be stuck or, in extreme
+          circumstances, split brained.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/etcdDivergentRevisions.md
+        summary: etcd is reporting divergent member revisions.
+      expr: |
+        -etcd_debugging_mvcc_compact_revision
+          + on (job, namespace) group_left (max_instance, max_pod)
```

Comment:
Here's an example of the rendered message from a pending alert in a cluster-bot cluster.

![image](https://user-images.githubusercontent.com/209920/160996499-2c85496b-1ef0-40f8-b85d-36112974e5c8.png)

Reply:
I've pushed 3bbdfdfc -> bd0a01ec to insert "trailing" between `instance` and `value`.  That should make it easier to find the value in the message; in the screenshot above^, my brain turns off when I hit the `instance` IP:port, and having a word before the `value` number would help it turn back on in time to actually read that number.

Reply:
@wking is "split brain" a technical term our support and customers will understand?

Reply:
[Our KCS Solution][1] is saying currently saying "data inconsistency" and [the upstream etcd annoucement][2] is currently saying "data corruption".  [etcd's upstream FAQ][3] includes

> ... changes must go through quorum to avoid membership configuration split brain.

I think it's important to distinguish "some member sticks, but the quorum guards either wedge etcd (quorum loss) or allow a reduced set of healthy members continue with quorum" from "competing members continue to accept writes and serve reads, even though they don't actually hold quorum anymore".  The former is bad.  The latter is terrible.  I'm not all that picky about which words get used, as long as divergent-quorum-ness is being expressed clearly.

[1]: https://access.redhat.com/solutions/6849521
[2]: https://etcd.io/docs/v3.5/op-guide/data_corruption/
[3]: https://etcd.io/docs/v3.5/faq/#why-doesnt-changing---listen-peer-urls-or---initial-advertise-peer-urls-update-the-advertised-peer-urls-in-etcdctl-member-list